src/ddoc/highlight.d

```diff
@@ -133,6 +133,28 @@ $(D_CODE $(D_KEYWORD unittest) {
 	assert(r1 == e1, r1);
 }
 
+unittest {
+	auto s = `
+        ---------
+        asm pure nothrow @nogc @trusted
+        {
+            // the compiler does not check the attributes
+            ret;
+        }
+        ---------
+`;
+	auto e = `
+        $(D_CODE $(D_KEYWORD asm) $(D_KEYWORD pure) $(D_KEYWORD nothrow) @nogc @trusted
```

Comment:
(Thanks for the edit ;) )
Nope, my expected output match yours. Indentation of comment is a separate matter, which I didn't actually try (as you can tell, my fix was motivated by the 'standalone' case).

Is this the result of a test ?